ui/public/i18n/en/translation.json

```diff
@@ -71,7 +71,9 @@
     "error_message_hostname": "By changing the user provider of an already configured NethVoice, all the users and extensions configuration will be reset.",
     "nethvoice_host_format": "Must be a valid domain name be a valid domain name.",
     "nethcti_ui_host_format": "Must be a valid domain name be a valid domain name.",
-    "set_rebranding": "Rebranding setup"
+    "set_rebranding": "Rebranding setup",
+    "proxy_not_installed": "Nethvoice Proxy",
+    "proxy_not_installed_description": "The proxy needs to be installed and configured before configuring NethVoice."
```

Comment:
```suggestion
    "proxy_not_installed_description": "The proxy needs to be installed before configuring NethVoice."
```

Reply:
It's better to make all the translations using weblate, and not in this PR?